src/web/server/htmlTemplate.ts

```diff
@@ -103,6 +103,40 @@ export const htmlTemplate = ({
 
     const twitterMetaTags = generateMetaTags(twitterData, 'name');
 
+    // Duplicated prefetch and preconnect tags from DCP:
+    // Documented here: https://github.com/guardian/frontend/pull/12935
+
+    // Information on preconnecting:
+    // https://css-tricks.com/using-relpreconnect-to-establish-network-connections-early-and-increase-performance/
+    const staticPreconnectUrls = [
+        `${CDN}`,
+        `https://i.guim.co.uk`,
+        `https://interactive.guim.co.uk`,
+        `https://www.google-analytics.com`,
+    ];
+
+    // Information on prefetching:
+    // https://developer.mozilla.org/en-US/docs/Web/Performance/dns-prefetch
+    const staticPrefetchUrls = [
+        `${CDN}`,
+        `https://i.guim.co.uk`,
+        `https://api.nextgen.guardianapps.co.uk`,
+        `https://hits-secure.theguardian.com`,
+        `https://j.ophan.co.uk`,
+        `https://ophan.theguardian.com`,
+        `https://phar.gu-web.net`,
+        `https://www.google-analytics.com`,
+        'https://sb.scorecardresearch.com',
```

Comment:
```suggestion
        `https://sb.scorecardresearch.com`,
```

just because all the others are

Reply:
That's odd, prettier should have picked that up